manifest.yaml

```diff
@@ -2,33 +2,37 @@ dist:
   module: github.com/open-telemetry/opentelemetry-collector-releases/core
```

Comment:
Could you change this to point to our repo?

Reply:
Done